SEIF-PROJ3-FE/src/DayCard/DayCard.jsx

```diff
@@ -1,27 +1,82 @@
-import { React, useState } from "react";
+import React, { useState, useEffect } from "react";
+import { getJournalEntryById } from "../service/journalentry";
+import JournalForm from "../JournalForm/JournalForm";
+
+function DayCard({ dateNo, day, journalEntryIds }) {
```

Comment:
As per discussed verbally, lets have it working for the full thing before we merge to main